perun-base/src/main/java/cz/metacentrum/perun/core/api/exceptions/FacilityMismatchException.java

```diff
@@ -0,0 +1,38 @@
+package cz.metacentrum.perun.core.api.exceptions;
+
+/**
+ * Exception thrown when wrong facility provided.
+ *
+ */
+public class FacilityMismatchException extends PerunException {
```

Comment:
might come in handy if this exception has facility as optional property for later usage